deepchem/models/torch_models/torch_model.py

```diff
@@ -539,7 +540,20 @@ def _predict(
       if self._use_openvino:
         output_values = [torch.Tensor(next(openvino_predictions))]
       else:
-        output_values = self.model(inputs)
+        # output_values = self.model(inputs)
+
+        # convert model
+        node_feats = inputs.ndata.pop('x')
+        edge_feats = inputs.edata.pop('edge_attr')
+        inp = [node_feats, edge_feats]
+
+        self.model.graph = inputs
+        # output_values = self.model(inp)
+
+        import io
+        buf = io.BytesIO()
+        torch.onnx.export(self.model, inp, buf, opset_version=11)
```

Comment:
If export finally works - please try to convert a model by MO locally. If it also works - let's move ONNX export to `deepchem/utils/openvino_model.py `